charts/incubator/shlink/values.yaml

```diff
@@ -0,0 +1,45 @@
+image:
+  repository: tccr.io/truecharts/shlink
+  pullPolicy: IfNotPresent
+  tag: v2.10.1@sha256:2abccbd1a150da840849f01789e4f824e35ee1136da7f5e56abaa86a290794ae
+
+service:
+  main:
+    protocol: HTTPS
+    ports:
+      main:
+        targetPort: 10153
+        port: 10153
+
+env:
+  PORT: 10153
+  DEFAULT_DOMAIN: ""
+  IS_HTTPS_ENABLED: true
+  GEOLITE_LICENSE_KEY: ""
+  DB_DRIVER: "postgres"
+  DB_USER: "{{ .Values.postgresql.postgresqlUsername }}"
+  DB_NAME: "{{ .Values.postgresql.postgresqlDatabase }}"
+
+envValueFrom:
+  DB_PASSWORD:
+    secretKeyRef:
+      name: dbcreds
+      key: postgresql-password
+  DB_HOST:
+    secretKeyRef:
+      name: dbcreds
+      key: plainhost
+  # REDIS_SERVERS:
+  #   secretKeyRef:
+  #     name: rediscreds
+  #     key: url
+
+postgresql:
+  enabled: true
+  existingSecret: "dbcreds"
+  postgresqlUsername: shlink
+  postgresqlDatabase: shlink
+
+# redis:
+#   enabled: true
+#   existingSecret: "rediscreds"
```

Comment:
```suggestion
  REDIS_SERVERS:
    secretKeyRef:
      name: rediscreds
      key: url

postgresql:
  enabled: true
  existingSecret: "dbcreds"
  postgresqlUsername: shlink
  postgresqlDatabase: shlink

redis:
  enabled: true
  existingSecret: "rediscreds"
  redisUsername: "shlink"
```

Reply:
You coiuld try an explicit username for redis.

It might also be caused by your explicitly adding the redis db int. (0) to the url. But that should not be the problem